docs/userguides/clis.md

```diff
@@ -100,13 +100,13 @@ Alternatively, you can call the `get_user_selected_account()` directly to have m
 
 ```python
```

Comment:
Still need to doctest our docs....

Reply:
This issue is tracked here: https://github.com/ApeWorX/ape/issues/657 and can be done any time, outside of a 0.7 release!